src/wot-adapter.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * WoT Adapter.ts
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.*
+ */
+
+import { AddonManagerProxy } from 'gateway-addon/lib/addon-manager-proxy';
```

Comment:
```suggestion
import { AddonManagerProxy, Adapter } from 'gateway-addon';
```
It won't work without the actual implementation.

Reply:
yeah right, I thought that the dependency was globally available for plugins, but checking other plugin implementations I saw that it was not the case. Thank you for spotting this early 👍🏻 . Resolving.